docs/components/alert/index.html

```diff
@@ -0,0 +1,39 @@
+<div class="alert-component">
+  <h1>Alert</h1>
+  <p class="text-lead margin-vertical-2">
+    Alert Component is used to show short messages or notifications based on the
+    user actions.
+  </p>
+  <p class="margin-vertical-3">
+    There are five types of alert components like <code>alert-primary</code>,
+    <code>alert-success</code>, <code>alert-info</code>,
+    <code>alert-warning</code> & <code>alert-danger</code>
+  </p>
+  <div class="alert alert-primary margin-vertical-2">
```

Comment:
The code looks good. I will suggest giving spacing between related things. Like here, you can give a space between `</p>` and `<div>`. It will increase the readability of the code.

Reply:
Okay Will keep that in mind.